src/MetricsReducer/SideBar/SideBar.tsx

```diff
@@ -218,7 +224,9 @@ export class SideBar extends SceneObjectBase<SideBarState> {
     const sectionValues = new Map();
 
     for (const filterKey of metricFiltersVariables) {
-      const filterValueFromUrl = urlSearchParams.get(filterKey);
+      // Check for both namespaced key (for embedded mode) and raw key (for regular mode)
+      const namespacedKey = `${embeddedTrailNamespace}-${filterKey}`;
```

Comment:
It'd be great to get this from `trail.state.urlNamespace` instead, but I couldn't figure out how to do that before the component is activated, which is too late.